src/frontend/src/app/pages/proposal-list/proposal-list.component.ts

```diff
@@ -0,0 +1,342 @@
+              >
+                {{ proposal.reviewPeriodEnd | formatDate }}
+              </app-value-col>
+
+              <app-key-col [id]="'proposal-voting-end-' + i">
+                Voting period end
+              </app-key-col>
+              <app-value-col
+                [attr.aria-labelledby]="'proposal-voting-end-' + i"
+              >
+                {{ proposal.votingPeriodEnd | formatDate }}
+              </app-value-col>
+
+              <app-key-col [id]="'proposal-date-decided-' + i">
+                Date decided
+              </app-key-col>
+              <app-value-col
+                [attr.aria-labelledby]="'proposal-date-decided-' + i"
+              >
+                {{
+                  proposal.decidedAt
+                    ? (proposal.decidedAt | formatDate)
+                    : 'Not yet decided'
+                }}
+              </app-value-col>
+
+              <app-key-col [id]="'proposal-codegov-vote-' + i">
+                CodeGov vote
+              </app-key-col>
+              <app-value-col
+                [attr.aria-labelledby]="'proposal-codegov-vote-' + i"
+                class="proposal__vote"
+                [ngClass]="{
+                  'proposal__vote--adopt': proposal.codeGovVote === 'ADOPT',
+                  'proposal__vote--reject': proposal.codeGovVote === 'REJECT'
+                }"
+              >
+                {{ proposal.codeGovVote }}
+              </app-value-col>
+            </app-key-value-grid>
+            <div class="btn-group">
+              <a
+                class="btn btn--outline"
+                [routerLink]="['/review', proposal.id, 'edit']"
+              >
+                My review
+              </a>
+              <a class="btn btn--outline" [routerLink]="['/open', proposal.id]">
+                View details
+              </a>
+            </div>
+          </div>
+        </cg-card>
+      }
+    }
+  `,
+})
+export class ProposalListComponent implements OnInit {
+  public proposalList$ = this.proposalService.currentProposalList$;
+
+  private selectedFilterSubject = new BehaviorSubject<
+    keyof typeof FilterOptions
+  >('InReview');
+  public selectedFilter$ = this.selectedFilterSubject.asObservable();
+
+  public readonly filterForm: FormGroup;
+  public readonly filterOptions = FilterOptions;
+  public readonly filterOptionsKeys = keysOf(FilterOptions);
+
+  public readonly proposalTopic = ProposalTopic;
+  public readonly linkBaseUrl = ProposalLinkBaseUrl;
+
+  private loadProposalsMessage: LoadingDialogInput = {
+    message: 'Loading data...',
+  };
+
+  constructor(
+    private readonly proposalService: ProposalService,
+    public readonly formBuilder: FormBuilder,
+    private readonly dialog: Dialog,
+  ) {
+    this.filterForm = this.formBuilder.group({
+      filter: ['InReview'],
+    });
+  }
+
+  public ngOnInit(): void {
+    this.selectedFilter$.subscribe(value => {
```

Comment:
If we use `this.filterForm.valueChanges` then there won't be a need to create a new observable just for the selected filter. Also this observable should `.pipe(untilDestroy())`.